pkgs/applications/virtualization/singularity/packages.nix

```diff
@@ -7,14 +7,14 @@ let
   apptainer = callPackage
     (import ./generic.nix rec {
       pname = "apptainer";
-      version = "1.2.2";
+      version = "1.2.4";
       projectName = "apptainer";
 
       src = fetchFromGitHub {
         owner = "apptainer";
         repo = "apptainer";
-        rev = "v${version}";
-        hash = "sha256-CpNuoG+QykP+HDCyFuIbZKYez5XnYrE75SWFoWu34rg=";
+        rev = "refs/tags/v${version}";
```

Comment:
Sorry for the noise, but why do we need the `refs/tags/` prefix?

I just noticed this when backporting an Apptainer security update (#319649).

Reply:
We don't "need" it per say, but it is recommended as it prevents any branch/tag ambiguity.